cms/djangoapps/contentstore/views/tests/test_import_export.py

```diff
@@ -428,6 +428,47 @@ def test_library_export(self):
         finally:
             shutil.rmtree(root_dir / name)
 
+    def test_library_import_then_export(self):
+        """
+        Verify that library exported after being imported first
+        """
+        library = LibraryFactory.create(modulestore=self.store)
+        lib_key = library.location.library_key
+        name = library.url_name
+
+        # import the library
+        extract_dir = path(tempfile.mkdtemp())
+        try:
```

Comment:
i am using the same directory for export below so add the finally block #468 for export_library_to_xml